src/Compilers/CSharp/Portable/Emitter/EditAndContinue/CSharpSymbolMatcher.cs

```diff
@@ -987,6 +995,11 @@ private bool AreTypesEqual(TypeSymbol type, TypeSymbol other)
                     members.AddRange(synthesizedMembers);
                 }
 
+                if (_otherDeletedMembers != null && _otherDeletedMembers.TryGetValue(symbol, out var deletedMembers))
```

Comment:
>_otherDeletedMembers [](http://example.com/codeflow?start=20&length=20)

nit:
`_otherDeletedMembers?.TryGetValue(symbol, out var deletedMembers) == true` avoid repeating `_otherDeletedMembers`. Same simplification can be applied to the previous if